### ISSUE TITLE
Not working with ECT templates after upgrading from 1.8.1

### ISSUE THREAD

**Issue by SteveMcArthur** [metalsmith/metalsmith-layouts#165]
I tried to move from 1.8.1 of the plugin to 2.2.0 and my existing projects using ECT templates broke. Created a bare bones simple project and still had the same problems. Consistently get the same error message:
`Cannot read property 'ect' of undefined in page on line`

Installed jstransformer-ect.

Is this me or is this a bug?
Thanks

**Comment by ismay** [comment]
It is a breaking change, so there is a chance that existing projects using v1 will break if you move them to v2. Please check out the readme for the current api.

If you need help try our slack channel.

**Comment by SteveMcArthur** [comment]
I actually downloaded the following project https://github.com/wernerglinka/ms-layout-njk which works fine with nunjucks. I tried swapping out nunjucks for ect (did all the necessary template changes and installed jstransformer-ect) and immediately got the `property 'ect' ` error. I trawled through the api docs, so unless I've missed something obvious or someone else has got ect to work, then I think this is a bug.
